src/data/pokemon-species-form.ts

```diff
@@ -208,6 +208,10 @@ export abstract class PokemonSpeciesForm {
     return this.baseStats[stat];
   }
 
+  /**
+   * Custom Sam logic to make Megas, Primals, and g-max Pokemon give 1.5x exp
```

Comment:
Let's not start a witch-hunt here.
```suggestion
   * Custom *legacy* logic to make Megas, Primals, and g-max Pokemon give 1.5x exp
```

Reply:
lol, didn't mean to witch hunt. Was mainly trying to date it to show that this has been around for 2 years

Reply:
I know, but we should avoid putting names into the code unless the person has given their consent